pkgs/CQLabs/homebrew-dcm/registry.yaml

```diff
@@ -22,8 +22,8 @@ packages:
             asset: dcm-{{.OS}}-release.exe.{{.Format}}
         supported_envs:
           - darwin
-          - windows
-          - amd64
+          - windows/amd64
+          - linux/amd64
```

Comment:
This change is unnecessary.

- windows/arm64 is supported by windows arm emulation. https://learn.microsoft.com/en-us/windows/arm/add-arm-support#emulation-on-arm-based-devices-for-x86-or-x64-windows-apps
- amd64 includes linux/amd64